provider/blob/src/export/iter.rs

```diff
@@ -0,0 +1,26 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::BlobDataProvider;
+use icu_provider::datagen::*;
+use icu_provider::prelude::*;
+
+impl DynamicDataProvider<BufferMarker> for BlobDataProvider {
```

Comment:
thought: this really is the same as `BufferProvider`

Reply:
I thought we had a blanket impl for this but I guess not